### PR TITLE
add `Discord_ClearPresence()`

### PR DESCRIPTION
**Pull request by snowyote**
send-presence example updated - start a line with 'c' to toggle
whether presence information is sent or not.

Added ClearPresence to .cs bindings

Added ClearPresence to UE4 blueprint class.

### CODE REVIEW

File: include/discord-rpc.h [hunk 1]
```diff
@@ -76,6 +76,7 @@ DISCORD_EXPORT void Discord_UpdateConnection(void);
 #endif
 
 DISCORD_EXPORT void Discord_UpdatePresence(const DiscordRichPresence* presence);
+DISCORD_EXPORT void Discord_ClearPresence();
```
Comment (Rapptz) [inline include/discord-rpc.h]: This is supposed to be `Discord_ClearPresence(void);`

Comment (snowyote) [reply include/discord-rpc.h]: Ah, good catch.  Will update on master.

**Comment by rojo8399** [comment]
Trying to build after this, but getting `nresolved external symbol __imp_Discord_ClearPresence referenced in function "public: void __cdecl UDiscordRpc::ClearPresence(void)" (?ClearPresence@UDiscordRpc@@QEAAXXZ)	Compile418	D:\Unreal Projects\Compile418\Intermediate\ProjectFiles\Module.discordrpc.cpp.obj	1	
`

**Comment by snowyote** [comment]
@rojo8399 It seems to work for me fine - I think you may need to build and then update your .lib and .dll in the example folder.  I've just added support to do this easily on master, just run `python build.py unreal` in the root of the repo and it should build and copy stuff where it needs to go.

**Pull request merged**